src/render.tsx

```diff
@@ -1,40 +1,59 @@
 import TestRenderer from 'react-test-renderer';
 import type { ReactTestInstance, ReactTestRenderer } from 'react-test-renderer';
 import * as React from 'react';
+import { Profiler } from 'react';
 import act from './act';
 import { addToCleanupQueue } from './cleanup';
 import debugShallow from './helpers/debugShallow';
 import debugDeep from './helpers/debugDeep';
 import { getQueriesForElement } from './within';
-import { setRenderResult } from './screen';
+import { setRenderResult, screen } from './screen';
+import { assertStringsWithinText } from './helpers/assertStringsWithinText';
 
 type Options = {
   wrapper?: React.ComponentType<any>;
   createNodeMock?: (element: React.ReactElement) => any;
+  experimentalValidateStringsRenderedInText?: boolean;
```

Comment:
Seems like React Test Render is using `unstable_` prefix for experimental options, seems like a good convention to follow:
```suggestion
  unstable_validateStringsRenderedInText?: boolean;
```
See:  [unstable_isConcurrent: boolean,](https://github.com/facebook/react/blob/cae635054e17a6f107a39d328649137b83f25972/packages/react-test-renderer/src/ReactTestRenderer.js#L64)